dept44-starter-scheduler/README.md

```diff
@@ -0,0 +1,140 @@
+# Dept44 Scheduler Starter
+
+A Spring Boot library that streamlines scheduled task management by offering robust exception handling, automated health
+checks, and schedlock integration.
+
+## Prerequisites
+
+- Java 21
+- Maven 3.9.9+
+- Spring Boot 3.x
+
+## Installation
+
+Include the dependency in your `pom.xml`:
+
+```xml
+
+<dependency>
+	<groupId>se.sundsvall.dept44</groupId>
+	<artifactId>dept44-scheduler</artifactId>
+	<version>6.0.6-SNAPSHOT</version>
+</dependency>
+```
+
+## Features
+
+1. **Automatic Health Tracking**  
+   Each scheduled task is registered with a dedicated health indicator. This allows real-time monitoring of task status
+   via the `/actuator/health` endpoint.
+
+2. **Centralized Exception Handling**  
+   Exceptions thrown during task execution are automatically logged and can restrict the system's health status to "
+   RESTRICTED."
+
+3. **ShedLock Integration**  
+   Ensures that only one instance of a task runs at a time across distributed systems.
+
+4. **Unique Request ID Assignment**  
+   Each task execution is assigned a unique `RequestId` for improved traceability in logs and monitoring tools.
+
+5. **Customizable Configurations**  
+   Scheduling, locking, and execution time limits are all configurable via properties or YAML files.
+
+6. **Manual Health Management**  
+   Allows for custom handling of exceptions within tasks while still enabling health status updates programmatically.
+
+## Usage
+
+1. **Enable the Scheduler**  
+   Create a configuration class to activate the scheduling aspect:
+
+   ```java
+   @Configuration
```

Comment:
Is this necessary when org.springframework.boot.autoconfigure.AutoConfiguration.imports is added to starter?

Reply:
I never got it to work without it before and found something about it being required for aspects to work. However I RTFM a bit closer and found that I hadn't set up AutoConfig correctly. Will adjust code and documentation